src/api/spec/models/obs_factory/distribution_strategy_opensuse_spec.rb

```diff
@@ -1,11 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe ObsFactory::DistributionStrategyOpenSUSE do
-  describe '#openqa_version' do
-    let(:strategy) { ObsFactory::DistributionStrategyOpenSUSE.new(project: create(:project, name: 'openSUSE:Leap:42.3')) }
+  let(:strategy) { ObsFactory::DistributionStrategyOpenSUSE.new(project: create(:project, name: 'openSUSE:Leap:42.3')) }
```

Comment:
:thought_balloon: can't you use `build(:strategy)` for this?

Reply:
I have a branch with factories for distribution and distribution strategy which would allow that. But I haven't finished it yet.
Once I got it to work, I'll update the existing code.